pymeasure/instruments/newport/esp300.py

```diff
@@ -203,11 +202,11 @@ def disable(self):
         """ Disables motion for the axis. """
         self.write("MF")
 
-    def home(self, type=1):
```

Comment:
It looks like the previous version of `home` was not well tested. Can you add `htype` to the docs, so that it is clear what this does?

Reply:
Honestly, I'm not completely clear on what the command does, so if I did add it to the documentation I would just be copying the manual. The change here just came from me realizing while looking through the code that an int wasn't passed where it clearly should be. I changed `type` to `htype` since `type` is a Python keyword, and although I'm sure Python can figure it out it seems like bad practice.

Reply:
Fair enough. Adding which values `htype` can take would be great. I think using `type` as a variable is acceptable, as long as you don't intend to request the type of a variable in the same local context (in this case the `home` method).